frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -597,7 +609,8 @@ class FilesApi extends ResourceApi<UFile, ProductStorage, UFileSpecification,
                 icon: "sensitivity",
                 enabled(selected, cb) {
                     const support = cb.collection?.status.resolvedSupport?.support;
-                    if ((support as FileCollectionSupport)) { /* ??? */}
+                    if ((support as FileCollectionSupport)) { /* ??? */
+                    }
```

Comment:
I think this is just supposed to be a type-assertion. This will probably work (please check if it compiles):
```suggestion
                    support as FileCollectionSupport;
```